visma/functions/constant.py

```diff
@@ -33,13 +38,156 @@ def differentiate(self):
         self.value = 0
 
     def integrate(self, intwrt):
-        self.coefficient = (self.value)**(self.power)
+        self.coefficient = self.value ** self.power
         self.__class__ = Variable
         self.power = [1]
         self.value = [intwrt]
 
+    def __radd__(self, other):
+        return self + other
+
+    def __add__(self, other):
+        if self.isZero():   # if one of them is Empty, we can return the other one even it is Empty too because we need at least one to be returned.
+            return other
+        elif other.isZero():
+            return self
+        elif isinstance(other, Constant):
+            const = Constant()
+            const.value = self.calculate() + other.calculate()
+            return const
+        elif isinstance(other, Expression):
+            if other.power == 1:
+                constFound = False
+                for i, var in enumerate(other.tokens):
+                    if isinstance(var, Constant):
+                        other.tokens[i] = self + var
+                        constFound = True
+                        break
+                if not constFound:
+                    other.tokens.extend(['+', self])
+                return other
+            else:
+                pass
+        self.value = self.calculate()
+        self.power = 1
+        self.coefficient = 1
+        exprAdd = Expression([self, '+', other])     # Make an Expression and assign the Tokens attribute with the Constant and the Other Variable, Trig. function,...etc.
+        return exprAdd
+
+    def __rsub__(self, other):
+        return self - other
+
+    def __sub__(self, other):
+        if self.isZero():
+            return -1 * other
+        elif other.isZero():
+            return self
+        elif isinstance(other, Constant):
+            const = Constant()
+            const.value = self.calculate() - other.calculate()
+            return const
+        elif isinstance(other, Expression):
```

Comment:
The above test case if updated will cover [this case](https://coveralls.io/builds/23068553/source?filename=visma%2Ffunctions%2Fconstant.py#L91).